packages/flutter/lib/src/cupertino/slider.dart

```diff
@@ -211,6 +211,11 @@ class CupertinoSlider extends StatefulWidget {
     properties.add(DoubleProperty('value', value));
     properties.add(DoubleProperty('min', min));
     properties.add(DoubleProperty('max', max));
+    properties.add(DiagnosticsProperty<ValueChanged<double>>('onChanged', onChanged));
```

Comment:
Similarly you will want
   properties.add(ObjectFlagProperty<ValueChanged<bool>>('onChanged', onChanged, ifNull: 'disabled')); for this one